task/net/src/main.rs

```diff
@@ -114,8 +114,9 @@ fn mac_address_from_uid(sys: &Sys) -> MacAddressBlock {
 
 #[cfg(feature = "vpd-mac")]
 fn mac_address_from_vpd() -> Option<MacAddressBlock> {
-    let i2c_task = I2C.get_task_id();
-    drv_local_vpd::read_config(i2c_task, *b"MAC0").ok()
+    use task_packrat_api::Packrat;
+    let packrat = Packrat::from(PACKRAT.get_task_id());
```

Comment:
Same question here – how does `net` know to wait until `packrat` has been loaded?

Reply:
It's implicit in the `BspImpl::preinit()` call where we wait for the sequencer - in the changes to both gimlet-seq and sidecar-seq, we should be loading packrat before transitioning to the state that releases net's `preinit()`.

The PSC didn't have a `preinit()` before this PR, but now it does and it works the same way: we wait until the psc-seq transitions to A2, which it does immediately after loading `packrat`.

Reply:
:+1: Added in 52ac248c